include/fastdds/dds/xtypes/dynamic_types/TypeDescriptor.hpp

```diff
@@ -215,7 +215,7 @@ class FASTDDS_EXPORTED_API TypeDescriptor
 
     /*!
      * Mofifies the is_nested property.
-     * @param[in] Boolean
+     * @param[in] is_nested
```

Comment:
```suggestion
     * @param[in] is_nested Boolean value to be set.
```

Reply:
Fixed in [8020bf2](https://github.com/eProsima/Fast-DDS/pull/4240/commits/8020bf2136b0ed793a5e093b26fc59595dbd7b26)